### DISCUSSION TITLE
Add ability to turn on flash

### DISCUSSION THREAD

**Discussion by gyroballmaster** [selfcustody/krux#217] [Ideas]
Since the device has an led on the back, being able to turn it on would be useful for scanning qr codes on paper in dim lit environments

**Comment by odudex** [comment]
You can turn it on, just press ENTER while the camera is active. Let us know if that works for you.

**Reply by gyroballmaster** [reply]
Ok Thanks, for the Amigo I was mainly using the touch screen for input. I pressed the Enter button, under the up and down button and it worked. Thanks!
